src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterEnterOnTokenTests.cs

```diff
@@ -481,7 +481,7 @@ void Method()
                 code,
                 'e',
                 indentationLine: 7,
-                expectedIndentation: 12);
+                expectedIndentation: 8);
         }
```

Comment:
@jcouv this branch has been failing for a couple of builds. I've fixed that test given that you've a prototype comment there.

Reply:
Thanks. Sorry about that (I've fixed this test in the recursive-patterns branch, but forgot to update the build-demo branch afterwards).